### ISSUE TITLE
PHP Warning:  Invalid argument supplied for foreach() in ... instaclick/medusa/src/Khepin/Medusa/Command/MirrorCommand.php on line 55

### ISSUE THREAD

**Issue by andygrunwald** [instaclick/medusa#16]
The error appears:
```
PHP Warning:  Invalid argument supplied for foreach() in /my/path/medusa/vendor/instaclick/medusa/src/Khepin/Medusa/Command/MirrorCommand.php on line 55
```

Log:
```
INFO: Checking out latest medusa configuration ...
fatal: BUG: get_tempfile_fd() called for inactive object
Copying satis-skeleton ...
Running medusa update ...
First getting all dependencies
PHP Warning:  Invalid argument supplied for foreach() in /my/path/medusa/vendor/instaclick/medusa/src/Khepin/Medusa/Command/MirrorCommand.php on line 55
- Getting dependencies for symfony/symfony
- Getting dependencies for monolog/monolog
- Getting dependencies for swiftmailer/swiftmailer
- Getting dependencies for symfony/icu
- Getting dependencies for symfony/intl
- Getting dependencies for psr/log
- Getting dependencies for twig/twig
- Getting dependencies for doctrine/common
- Getting dependencies for doctrine/annotations
- Getting dependencies for doctrine/lexer
- Getting dependencies for doctrine/collections
- Getting dependencies for doctrine/cache
- Getting dependencies for doctrine/inflector
- Getting dependencies for doctrine/orm
- Getting dependencies for symfony/console
- Getting dependencies for doctrine/instantiator
- Getting dependencies for doctrine/dbal
- Getting dependencies for twig/extensions
- Getting dependencies for symfony/assetic-bundle
```

**Comment by andygrunwald** [comment]
This errors only occur if you don`t have the `` key in your medusa.json:

```
{
    "repositories": [],
    "require": [
        "symfony/symfony",
        "monolog/monolog",
        "swiftmailer/swiftmailer",
        "symfony/icu",
        "symfony/intl",
        "psr/log",
        "twig/twig",
        ...
    ],
    "repodir": "/.../git-mirror",
    "satisurl": "http://...",
    "satisconfig": "satis.json"
}
```

If you include this as an empty array, everything is fine: `"repositories": []`

**Comment by andygrunwald** [comment]
PR to fix this is in #24
